src/__tests__/03-create-channels.ts

```diff
@@ -115,16 +119,30 @@ it('opens private channel from lndOutside1 to lndOutside2', async () => {
 })
 
 it('opens channel from lndOutside1 to lnd1', async () => {
-  const socket = `lnd:9735`
-  await openChannelTesting({ lnd: lndOutside1, other_lnd: lndMain, socket })
+  const socket = `lnd1:9735`
+  await openChannelTesting({ lnd: lndOutside1, other_lnd: lnd1, socket })
 
   {
-    const { channels } = await getChannels({ lnd: lndMain })
+    const { channels } = await getChannels({ lnd: lnd1 })
     expect(channels.length).toEqual(channelLengthMain + 1)
   }
 
 })
 
+it('opens channel from lnd1 to lnd2', async () => {
+  const socket = `lnd2:9735`
+  await openChannelTesting({ lnd: lnd1, other_lnd: lnd2, socket })
+
+  const { channels } = await getChannels({ lnd: lnd1 })
+  expect(channels.length).toEqual(channelLengthMain + 1)
+  
+  const channel = _.find(channels, {partner_public_key: getLnds()[1].pubkey})
+  const input = {fee_rate: 0, base_fee_tokens: 0, transaction_id: channel!.transaction_id, transaction_vout: channel!.transaction_vout}
+
+  await updateRoutingFees({lnd: lnd1, ...input});
```

Comment:
Why is this needed?

Reply:
we want to be able to receive on-us transaction even if one of the 2 nodes we have is offline.

ie: 
1/ user A show a QR code from node 1 and send the invoice to someone else.
2/ node 1 goes offline for whatever reason
3/ user B pays the invoice. because the invoice is part of the `isMyNode()` set, basically it settle offline, with 0 fee, even if one node generated the invoice and another node would have paid for it through a common channel.

only caveat here is that graphql would not delete the invoice from node A

and so because it settle in layer 3 "off lightning", there is no fee currently. so I wanted to have the same no-fee connection between 2 nodes of the same group. 

now that I think about it, this is not really necessary, and we could still have fees between those.

Reply:
> only caveat here is that graphql would not delete the invoice from node A

Is that a problem? (Apart from just slightly bloating the db)

> now that I think about it, this is not really necessary, and we could still have fees between those.

Yeah, there will be 0 fees irrespective of the fees set for the channel because they're directly connected.